camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -206,8 +206,10 @@
                       <v-select
                         :label="selectSearchType"
                         :options="reviewerUsersList"
+                        :filterable="false"
                         v-model="userSelected"
-                        placeholder="Search by Lastname"
+                        :placeholder="`Search by ${selectSearchType}`"
```

Comment:
I believe,  `""` is not required, can you check it?

Reply:
It checked it. It's required

![image](https://user-images.githubusercontent.com/70306694/134504258-ad5732fa-3240-4a3a-9762-745984373ce1.png)